python/pyspark/pandas/tests/test_ops_on_diff_frames.py

```diff
@@ -503,6 +503,12 @@ def test_loc_getitem_boolean_series(self):
             (pdf1.A + 1).loc[pdf2.A > -3].sort_index(), (psdf1.A + 1).loc[psdf2.A > -3].sort_index()
         )
 
+        pser = pd.Series([0, 1, 2, 3, 4], index=[20, 10, 30, 0, 50])
+        psser = ps.from_pandas(pser)
+        self.assert_eq(pser.loc[pdf2.A > -3].sort_index(), psser.loc[psdf2.A > -3].sort_index())
+        pser.name = psser.name = "B"
+        self.assert_eq(pser.loc[pdf2.A > -3].sort_index(), psser.loc[psdf2.A > -3].sort_index())
```

Comment:
Could you also try with Series on the same anchor?

Reply:
Certainly, added `IndexingTest.test_loc_getitem_boolean_series`